pkg/descheduler/pod/pods_test.go

```diff
@@ -29,6 +30,31 @@ import (
 	"sigs.k8s.io/descheduler/test"
 )
 
+var (
+	lowPriority  = int32(0)
+	highPriority = int32(10000)
+)
+
+func setHighPriority(pod *v1.Pod) { pod.Spec.Priority = &highPriority }
+func setLowPriority(pod *v1.Pod)  { pod.Spec.Priority = &lowPriority }
+
+func makeBestEffortPod(pod *v1.Pod) {
+	pod.Spec.Containers[0].Resources.Requests = nil
+	pod.Spec.Containers[0].Resources.Requests = nil
+	pod.Spec.Containers[0].Resources.Limits = nil
+	pod.Spec.Containers[0].Resources.Limits = nil
+}
+
+func makeBurstablePod(pod *v1.Pod) {
+	pod.Spec.Containers[0].Resources.Limits = nil
+	pod.Spec.Containers[0].Resources.Limits = nil
+}
+
+func makeGuaranteedPod(pod *v1.Pod) {
+	pod.Spec.Containers[0].Resources.Limits[v1.ResourceCPU] = pod.Spec.Containers[0].Resources.Requests[v1.ResourceCPU]
+	pod.Spec.Containers[0].Resources.Limits[v1.ResourceMemory] = pod.Spec.Containers[0].Resources.Requests[v1.ResourceMemory]
+}
+
```

Comment:
@lixiang233 yes, in my opinion I think it would be good to have `makeGuaranteedPod`, `makeBurstablePod` and `makeBestEffortPod` in `test/test_utils.go`